crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -516,6 +516,28 @@ declare_lint_rule! {
 }
 ```
 
```

Comment:
Can you update the table of contents at the very beginning and include this new header?